doc/installation/linux_install.rst

```diff
@@ -129,3 +117,40 @@ If you want to run the ``nest`` executable or use the ``nest`` Python module wit
 The script ``/install/path/bin/nest_vars.sh`` can be sourced in ``.bashrc`` and will set these paths for you. This also allows to switch between NEST installations in a convenient manner.
 
 
+Environment variables
+---------------------
+
+There are several environment variables that describe where components of the NEST installation can be found. In particular when installing to a custom directory, it is typically necessary to explicitly set these variables, so that your operating system can find the NEST binaries and libraries.
+
+A shell script is provided in ``</install/path/>bin/nest_vars.sh`` to make setting the environment variables more convenient. Setting the environment variables in your active shell session requires sourcing the script:
+
+.. code-block:: sh
+
+   source </install/path/>bin/nest_vars.sh
+
+You may want to include this line in your ``.bashrc`` file, so that the environment variables are set automatically.
```

Comment:
```suggestion
You may want to include this line in your ``.bashrc`` file, so that the environment variables are set automatically whenever you open a new terminal.
```